### ISSUE TITLE
Unable to explore a pandas DataFrame

### ISSUE THREAD

**Issue by lodagro** [kylepollina/objexplore#10]
**Describe the bug**

A ValueError is raised when exploring a pandas DataFrame, which is related to ```objexplore.utils.is_selectable```.
```python
In [1]: import objexplore                                                      
In [2]: import pandas as pd                                                    
In [3]: df = pd.DataFrame()                                                    
In [4]: objexplore.utils.is_selectable(df)      
```

**Traceback**
```
ValueError                                Traceback (most recent call last)
<ipython-input-4-b894c6ec269f> in <module>
----> 1 objexplore.utils.is_selectable(df)

<...>/lib/python3.7/site-packages/objexplore/utils.py in is_selectable(obj)
      1 def is_selectable(obj):
----> 2     return obj not in (None, [], (), {}, set())

<...>/lib/python3.7/site-packages/pandas-1.0.5-py3.7-linux-x86_64.egg/pandas/core/generic.py in __nonzero__(self)
   1477     def __nonzero__(self):
   1478         raise ValueError(
-> 1479             f"The truth value of a {type(self).__name__} is ambiguous. "
   1480             "Use a.empty, a.bool(), a.item(), a.any() or a.all()."
   1481         )

ValueError: The truth value of a DataFrame is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all().

```

**Comment by kylepollina** [comment]
Thanks for the report, fix is in v1.4.4 - upgrade with `pip install objexplore --upgrade` and see if the bug persists.

Thank you!

**Comment by lodagro** [comment]
Works fine now. Thanks.
